tests/api/circulation/test_loan_bulk_extend.py

```diff
@@ -22,32 +22,45 @@
 from tests.api.conftest import _create_records
 from tests.helpers import user_login
 
-pid_start_value = 99
-amount_of_ongoing_loans = 5
-pid_end_value = pid_start_value + amount_of_ongoing_loans
+pid_start_value = 200
+amount_of_ongoing_loans = 3
+amount_of_overdue_loans = 3
+pid_end_value = \
+    pid_start_value + amount_of_ongoing_loans + amount_of_overdue_loans
 
 
 def create_loans(start=pid_start_value, end=pid_end_value):
     ongoing_loans = []
+    overdue_loans = []
 
     day_range_loan_extendable = \
         current_app.config["ILS_CIRCULATION_LOAN_WILL_EXPIRE_DAYS"]
 
     now = arrow.utcnow()
-    end_date_not_expired = now + timedelta(days=day_range_loan_extendable - 1)
 
-    def loan_generator(pid_value):
+    start_date_expiring = now.isoformat()
```

Comment:
what do you mean by expiring ?

Reply:
overdue = passed the loan end date already
expiring = soon to pass the end date (depending on config)